DESCRIPTION

```diff
@@ -28,6 +28,8 @@ Imports:
     ggrepel,
     git2r,
     glmnet,
+    grid,
```

Comment:
again, check whether we really need dependencies.

Reply:
sorry, don't know how this package made it into the library, but i guess the other two pcks `gridExtra` and `RCurl` are ok, as they could be both usefull for other articles too?

Reply:
yeah